### ISSUE TITLE
ImportError: cannot import name 'stream_fastq_full'

### ISSUE THREAD

**Issue by george-githinji** [wdecoster/fastqmetrics#1]
On `Python 3.6`, `OS X 10.12.3` after running 
```
pip3 install fastqmetrics
Collecting fastqmetrics
  Downloading fastqmetrics-0.1.0.tar.gz
Collecting nanoget>=0.14.0 (from fastqmetrics)
  Downloading nanoget-0.14.0.tar.gz
Requirement already satisfied: pandas in /usr/local/lib/python3.6/site-packages (from nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: numpy in /usr/local/lib/python3.6/site-packages (from nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: biopython in /usr/local/lib/python3.6/site-packages (from nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: pysam>0.10.0.0 in /usr/local/lib/python3.6/site-packages (from nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: nanomath in /usr/local/lib/python3.6/site-packages (from nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: pytz>=2011k in /usr/local/lib/python3.6/site-packages (from pandas->nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: python-dateutil>=2 in /usr/local/lib/python3.6/site-packages (from pandas->nanoget>=0.14.0->fastqmetrics)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.6/site-packages (from python-dateutil>=2->pandas->nanoget>=0.14.0->fastqmetrics)
Building wheels for collected packages: fastqmetrics, nanoget
  Running setup.py bdist_wheel for fastqmetrics ... done
  Stored in directory: /Users/username/Library/Caches/pip/wheels/5f/dc/83/88496d60b9a41ff9735641f32abe35bb4959f62a15fb292001
  Running setup.py bdist_wheel for nanoget ... done
  Stored in directory: /Users/username/Library/Caches/pip/wheels/fa/83/3a/68d4da094ce7e51c7fab77f87becd310f79b9f63a55627cf04
Successfully built fastqmetrics nanoget
Installing collected packages: nanoget, fastqmetrics
  Found existing installation: nanoget 0.13.1
    Uninstalling nanoget-0.13.1:
      Successfully uninstalled nanoget-0.13.1
Successfully installed fastqmetrics-0.1.0 nanoget-0.14.0
```
The attempting to run the tool, with 
```
fastqmetrics --threads 2 ../demultiplexed_run1/data/BC10.fastq
Traceback (most recent call last):
  File "/usr/local/bin/fastqmetrics", line 7, in <module>
    from fastqmetrics.fastqmetrics import main
  File "/usr/local/lib/python3.6/site-packages/fastqmetrics/fastqmetrics.py", line 1, in <module>
    from nanoget import stream_fastq_full
ImportError: cannot import name 'stream_fastq_full'
```

**Comment by wdecoster** [comment]
Thanks for reporting! 

Oops, that was a little oversight. I changed the name of the function after pushing to pypi, apparently. Stupid. I just pushed nanoget 0.14.1 to pypi which should contain the correct function name `stream_fastq_full`. Updating using `pip install nanoget --upgrade` should do the trick...

**Comment by george-githinji** [comment]
Thank you. But now we have a new one, 

```
readname	length	averageQ	medianQ
multiprocessing.pool.RemoteTraceback:
"""
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/usr/local/lib/python3.6/site-packages/nanoget/nanoget.py", line 330, in extract_all_from_fastq
    nanomath.median_qual(rec.letter_annotations["phred_quality"]))
AttributeError: module 'nanomath' has no attribute 'median_qual'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/fastqmetrics", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/fastqmetrics/fastqmetrics.py", line 9, in main
    for res in stream_fastq_full(args.fastq, args.threads):
  File "/usr/local/lib/python3.6/site-packages/nanoget/nanoget.py", line 304, in stream_fastq_full
    for results in pool.imap(extract_all_from_fastq, SeqIO.parse(inputfastq, "fastq")):
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/multiprocessing/pool.py", line 735, in next
    raise value
AttributeError: module 'nanomath' has no attribute 'median_qual'
```

**Comment by wdecoster** [comment]
Is nanomath the latest version? Updating could fix this. I have been a bit sloppy with specifying the required version it seems. Apologies!

**Comment by george-githinji** [comment]
Yes nanomath is the latest version. 

```
pip3 install  nanomath --upgrade
Requirement already up-to-date: nanomath in /usr/local/lib/python3.6/site-packages
Requirement already up-to-date: pandas in /usr/local/lib/python3.6/site-packages (from nanomath)
Requirement already up-to-date: numpy>1.8 in /usr/local/lib/python3.6/site-packages (from nanomath)
Requirement already up-to-date: pytz>=2011k in /usr/local/lib/python3.6/site-packages (from pandas->nanomath)
Requirement already up-to-date: python-dateutil>=2 in /usr/local/lib/python3.6/site-packages (from pandas->nanomath)
Requirement already up-to-date: six>=1.5 in /usr/local/lib/python3.6/site-packages (from python-dateutil>=2->pandas->nanomath)

```

**Comment by wdecoster** [comment]
Can you please tell me the version number too?
`python - c "import nanomath ; print(nanomath.__version__)" `

**Comment by george-githinji** [comment]
This what I have;
```
python3 -c "import nanomath; print(nanomath.__version__)"
0.13.0
```

**Comment by wdecoster** [comment]
Thanks for the information, I'll fix this when I get at work, in a few hours.

**Comment by wdecoster** [comment]
I think this should be fixed with nanomath 0.13.1

**Comment by wdecoster** [comment]
I'll consider this solved - please let me know if you experience further issues.
